crates/bevy_time/src/fixed_timestep.rs

```diff
@@ -42,6 +49,7 @@ impl FixedTime {
         FixedTime {
             accumulated: Duration::ZERO,
             period,
+            max_ticks_per_frame: Some(3),
```

Comment:
There should be a `const` for this default value so future contributors don't forget to update one of these constructors. I could see this easily slipping past reviewers.